products/salt.md

```diff
@@ -0,0 +1,67 @@
+---
+
+title: Salt
+category: server-app
+tags: salt
```

Comment:
```suggestion
tags: python-runtime
```

We're using a limited set of tags a favor reusable ones, cf. tags on https://endoflife.date/ansible.